docs/python/api_summary.rst

```diff
@@ -37,7 +37,30 @@ the kernel is executed on CPU.
 		model, providers=['CUDAExecutionProvider', 'CPUExecutionProvider']
 	)
 
-The list of available execution providers can be found here: `Execution Providers <https://onnxruntime.ai/docs/execution-providers>`_.
+Available execution providers:
+- `CPUExecutionProvider`
+- `CUDAExecutionProvider`
+- `TensorrtExecutionProvider`
+- `OpenVINOExecutionProvider`
+- `DmlExecutionProvider`
+- `QNNExecutionProvider`
+- `NnapiExecutionProvider`
+- `CoreMLExecutionProvider`
+- `XnnpackExecutionProvider`
+- `ROCMExecutionProvider`
+- `MIGraphXExecutionProvider`
+- `VitisAIExecutionProvider`
+- `AzureExecutionProvider`
+- `ACLExecutionProvider`
+- `ArmNNExecutionProvider`
+- `CANNExecutionProvider`
+- `TvmExecutionProvider`
+- `VSINPUExecutionProvider`
+- `JsExecutionProvider`
+- `RknpuExecutionProvider`
+- `WebNNExecutionProvider`
+
+For more information, see [Execution Providers](https://onnxruntime.ai/docs/execution-providers/).
```

Comment:
why not just refer to [`get_available_providers`](https://onnxruntime.ai/docs/api/python/api_summary.html#onnxruntime.get_available_providers)?

if we list them here, we need to keep this list up to date. the actual available EPs can vary from version to version.

Reply:
agreed. this is also what I had responded for https://github.com/microsoft/onnxruntime/issues/22101#issuecomment-2357667377